### PR TITLE
Update to 4.6.20156

### PR DESCRIPTION
**Pull request by glitchcrab**
towards giantswarm/giantswarm/issues/11449

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+### Changed
+
+## [v4.6.0] 2020-06-09
+
+### Changed
+
+- Updated image versions from `4.5.20069` to `4.6.20156`.
+
+[Unreleased]: https://github.com/giantswarm/nginx-ingress-controller-app/compare/v4.6.0...master
+[v4.6.0]: https://github.com/giantswarm/nginx-ingress-controller-app/releases/tag/v4.6.0
```
Comment (jgsqware) [inline CHANGELOG.md]: nginx?

Comment (glitchcrab) [reply CHANGELOG.md]: copy paste fail :facepalm:

**Comment by glitchcrab** [comment]
ping @jgsqware again

**Review by sslavic — APPROVED** [review]
LGTM

**Review by yasn77 — APPROVED** [review]
@glitchcrab 

Since the image tag is the same for each component (and I doubt that will ever be different). Is it worth consolidating them as image tag reference? Saves having to change it multiple times. Or use YAML anchor?

Just a thought.

**Review by jgsqware — APPROVED** [review]
LGTM

**Pull request merged**